borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+`-webkit-app-region: drag;` will be enabled.
+
+### 3. CSS display-mode media query for borderless
+
+Similarly to other display modes, borderless should be queryable with @media.
+
+Example media query with borderless:
+
+```
+@media (display-mode: borderless) {
+  .any-css-class-name {
+    margin: 5px;
+  }
+}
+```
+
+See:
+[https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode)
+
+### 4. Leveraging Window Management permission for borderless
+
+In order for the borderless mode to activate, the app needs to have Window
+Management permission granted. The permission prompt can be triggered using
+JavaScript.
+
+For the **managed** context, permission prompts can be by-passed with a policy
+set by an admin and later on edited in the app’s settings. For the **unmanaged**
+context, the Window management API permission must be granted.
+
+#### Enabling Window Management
+
+Granting the window management API permission can be done with the following
+JavaScript:
+
+```
+const screenDetails = await window.getScreenDetails();
+```
+
+That will show the permission prompt to enable Window Management API permission
+which will also enable borderless capability in case the permission is given.
+
+Later on, the state of the permission can be queried with the following script:
+
+```
+navigator.permissions.query({name:'window-placement'})
+  .then((status) => {
+    // Do what you need with the permission state.
+    console.log(status.state)
+  });
+```
+
+#### Disabling Window Management
+
+To disable borderless mode, the user can disable the Window Management
+permission in App Settings.
+
+In case the Window management permission is not provided or no longer in place,
+the app will follow the normal display mode hierarchy based on the
+`display_override` and display values on the `manifest.json`. In case no other
+display mode is provided, it will
+[default to standalone mode](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/web_applications/web_app_proto_utils.cc;drc=4a8573cb240df29b0e4d9820303538fb28e31d84;l=161)
+similarly to WCO and other `display_override` modes.
+
+### 5. Isolated Web App check
+
+Borderless mode is only available for
+[Isolated Web Apps (IWAs)](https://github.com/WICG/isolated-web-apps).
+
+## Related features
+
+- [app-details feature](crbug.com/1225871) (Googlers-only)
+- [go/cros-privacy-indicators-design](go/cros-privacy-indicators-design)
+  (Googlers-only)
+- [additional windowing controls](https://github.com/ivansandrk/additional-windowing-controls/blob/main/awc-explainer.md)
+
+### Displaying app’s origin in the app settings UI
+
+Since without the title bar and the 3-dot menu, the app’s origin won’t be visible
+anymore, it needs to be shown somewhere, in this case in App Settings. This
```

Comment:
The general discussion here is good (about the security need to display the app's origin), but the explainer should not be specific about where it goes in ChromeOS UI, because this document is not a ChromeOS document.

"In this case in App Settings" and "This work has been completed" should be removed; instead, talk about the general problem and how a hypothetical user agent might solve it. You might say "On ChromeOS, for example, we intend to put this in the OS settings panel for the app", but only use it as an example, and do not assume the reader knows about OS-specific UI like "App Settings".